cartography/intel/azure/__init__.py

```diff
@@ -58,7 +58,7 @@ def _sync_one_subscription(
     config: Config,
 ) -> None:
 
-    common_job_parameters['Azure_Active_Directory_Name'] = tenant['defaultDomain']
+    common_job_parameters['Azure_Primary_AD_Domain_Name'] = tenant['defaultDomain']
```

Comment:
i don't see code to delete `common_job_parameters['Azure_Primary_AD_Domain_Name']` anywhere? We should clean it up after processing of the consolelinks everywhere.